README.md

```diff
@@ -78,6 +83,8 @@ Supported methods:
 - [x] [SPIN](https://www.seas.upenn.edu/~nkolot/projects/spin/) (ICCV'2019)
 - [x] [VIBE](https://github.com/mkocabas/VIBE) (CVPR'2020)
 - [x] [HybrIK](https://jeffli.site/HybrIK/) (CVPR'2021)
+- [x] [PARE](https://pare.is.tue.mpg.de/) (ICCV'2021)
+- [x] [DeciWatch](https://arxiv.org/pdf/2203.08713.pdf)
```

Comment:
Should we add `(arXiv'2022)`? And the link should direct to [homepage](https://ailingzeng.site/deciwatch) if it exists.